go.mod

```diff
@@ -201,6 +201,13 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require (
+	github.com/google/pprof v0.0.0-20230111200839-76d1ae5aea2b
+	github.com/richardartoul/molecule v1.0.1-0.20221107223329-32cfee06a052
+	github.com/spaolacci/murmur3 v1.1.0
+	github.com/zeebo/xxh3 v1.0.2
+)
```

Comment:
Can you bring these in to the main `require` block and re-run `go mod tidy` so we don't have more than two require blocks?